listener_manager.go

```diff
@@ -62,11 +62,12 @@ func (m *ListenerManager) announceStatus(status *PNStatus) {
 	go func() {
 		m.RLock()
 		m.pubnub.Config.Log.Println("announceStatus lock")
+	B:
 		for l := range m.listeners {
 			select {
 			case <-m.exitListener:
 				m.pubnub.Config.Log.Println("announceStatus exitListener")
-				break
+				break B
```

Comment:
Can we use a different label here that is more clear?

Reply:
Modified to AnnounceStatusLabel / AnnounceMessageLabel